odoo/addons/base/models/ir_cron.py

```diff
@@ -465,17 +465,13 @@ def _run_job(cls, job) -> CompletionStatus:
                     else:
                         status = CompletionStatus.FAILED
                 else:
-                    if not progress.remaining:
-                        status = CompletionStatus.FULLY_DONE
-                    elif not progress.done:
-                        # assume the server action doesn't use the progress API
-                        # and that there is nothing left to process
-                        status = CompletionStatus.FULLY_DONE
-                    else:
-                        status = CompletionStatus.PARTIALLY_DONE
 
-                    if status == CompletionStatus.FULLY_DONE and progress.deactivate:
-                        job['active'] = False
+                    if progress.remaining:
+                        status = CompletionStatus.PARTIALLY_DONE
```

Comment:
"cron done=0 remaining=1 should be partially done" - in theory yes, but given the current implementation there are some drawbacks.

"Partially done" means we continue the loop, but the job did nothing - so the probability it will just continue doing nothing is high. Some examples: time limit reached, all records to process are locked, some resource is not available, etc. In case of done=0, we should break the main loop - adding here `if not progress.done: break`. For readability, all breaking could be done before the `finally` statement.

It might be good to set MIN_RUNS_PER_JOB=1 (read: remove it). There is already a time-limit and I don't see the point of an arbitrary limit.

(BTW, this commit should be in another PR)